app/lib/simple_layout.rb

```diff
@@ -308,9 +308,8 @@ def setup_main_layout!
 
   private
 
-  def forum_builtin_static_pages
-    return [] unless provider.provider_can_use?(:forum)
-
+  def forum_builtin_static_pages # rubocop:disable Metrics/MethodLength
+    # TODO: clean up
```

Comment:
Why leave this method?
Just assuming that this is left as a reference?

Reply:
I will remove "builtin pages" stuff later (methods, events, tests, etc)